Packages/cdms2/Src/Cdunifmodule.c

```diff
@@ -1048,6 +1048,9 @@ cdunif_type_from_type(char array_type)
   case NPY_DOUBLE:
     type = NC_DOUBLE;
     break;
+  case NPY_STRING:
+    type = NC_STRING;
+    break;
   default:
```

Comment:
In one of them I forgot the "break" before the default.  You want the 
one with "break".

I think both NPY_STRING case can be match to NC_STRING.


Denis


On 5/25/16 3:32 PM, Charles Doutriaux wrote:
>
> In Packages/cdms2/Src/Cdunifmodule.c 
> <https://github.com/UV-CDAT/uvcdat/pull/1984#discussion_r64665023>:
>
> > @@ -1048,6 +1048,9 @@ cdunif_type_from_type(char array_type)
> >    case NPY_DOUBLE:
> >      type = NC_DOUBLE;
> >      break;
> > +  case NPY_STRING:
> > +    type = NC_STRING;
> > +    break;
> >    default:
>
> in your other branch you have
>
> |+ case NPY_STRING: + case NPY_STRINGLTR: + type = NC_STRING; |
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/UV-CDAT/uvcdat/pull/1984/files/43b86b09024b2ffed36b20ebbf08dc6c333492c2#r64665023>
>